main.go

```diff
@@ -110,25 +230,60 @@ func main() {
 	filePath := os.Args[1]
 	debug("## Loading " + filePath)
 
-	fileData, err := ioutil.ReadFile(filePath)
+	var subsetPartition map[int][]int
+	err, test := readTestFile(filePath)
 	if err != nil {
 		fatal(err)
 	}
 
-	var test Test
-	var summary Summary
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
yeah better put this inside an `init()` function


https://medium.com/golangspec/init-functions-in-go-eac191b3860a